projects/core/components/primitive-textfield/primitive-textfield.style.less

```diff
@@ -111,3 +111,11 @@
     position: relative;
     pointer-events: none;
 }
+
+.t-value-decoration {
```

Comment:
@waterplea without it, the text ellipsis just appears, although the text of the t-filler is hidden, but the ellipsis is shown, is this a browser bug?

Reply:
Got it